scripts/components/networking/statssynchronizercomponent/StatsSynchronizerComponent.gd

```diff
@@ -159,13 +160,13 @@ var is_dead: bool = false
 var server_buffer: Array[Dictionary] = []
 var ready_done: bool = false
 
-var _default_hp_max: int = hp_max
-var _default_energy_max: int = energy_max
-var _default_energy_regen: int = energy_regen
-var _default_attack_power_min: int = attack_power_min
-var _default_attack_power_max: int = attack_power_max
-var _default_defense: int = defense
-var _default_movement_speed: float = movement_speed
+var _default_hp_max: int = 100
```

Comment:
why would you set these to hardcoded values and not to the values of the exports?

Reply:
Because the defaults are meant to be a reliable base.  
Having the defaults depend on the current value means that if a player where to be loaded with different values, then these would become the new defaults. Making it impossible for the StatsSynchronizer to return to the correct amount even after calling load_defaults().  
  
I can return it to how it was and make a work around, but the risk of the defaults changing would still be present. Possibly requiring admin intervention to fix a player's stats

Reply:
You're right, the defaults should be a fixed value and don't depend on changeable variables. However I do think they should be configurable.

Reading this, we should probably export the default values, making them configurable for each entity using this component.